.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+name: Snuba Checks
+on:
+  push:
+    branches:
+      - master
+      - releases/**
+  pull_request:
```

Comment:
Not very familiar with GHA, but does this work with PR coming from forks? We had issues with google cloud build there, it is important that use case is taken into account for external contributors.

Reply:
I've addressed this.